Content/GUI/Quests/QuestsUIState.cs

```diff
@@ -17,28 +40,64 @@ public void Toggle()
 		if (IsVisible)
 		{
 			IsVisible = false;
+			_questDetails.Remove();
+			Visible = false;
```

Comment:
Visible is a `get;` only I think so I'm not sure how this is being set, but it points to IsVisible. Is there a reason for setting both?